app/src/main/java/org/sopt/pingle/presentation/ui/joingroup/JoinGroupCodeActivity.kt

```diff
@@ -0,0 +1,34 @@
+package org.sopt.pingle.presentation.ui.joingroup
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.sopt.pingle.R
+import org.sopt.pingle.databinding.ActivityJoinGroupCodeBinding
+import org.sopt.pingle.util.base.BindingActivity
+
+@AndroidEntryPoint
+class JoinGroupCodeActivity :
+    BindingActivity<ActivityJoinGroupCodeBinding>(R.layout.activity_join_group_code) {
+    private val joinViewModel by viewModels<JoinViewModel>()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        addObservers()
+    }
+
+    private fun addObservers() {
+        joinViewModel.run {
+            joinGroupData.observe(this@JoinGroupCodeActivity) {
```

Comment:
```suggestion
            joinGroupData.observe(this@JoinGroupCodeActivity) { joinGroupData ->
```
이런 식으로 작성해서 it 대신 사용해주면 조금 더 가독성이 좋아지겠죵?

Reply:
그총 람다에는 네임드 해주면 좋습니다